cpp/src/groupby/sort/group_single_pass_reduction_util.cuh

```diff
@@ -124,7 +153,7 @@ struct null_replaced_value_accessor : value_accessor<T> {
 template <aggregation::Kind K>
 struct reduce_functor {
   template <typename T>
-  static constexpr bool is_supported()
+  static constexpr bool is_natively_supported()
```

Comment:
> What does "natively" supported mean that "is_supported" didn't mean?

Just `is_supported` means we can proceed with the data, otherwise we "don't support such data and will throw exception". Because `is_supported(struct) => false` but we still accept the input data, I have to change `is_supported` to `is_natively_supported` and recently I just changed again to `is_trivially_supported` (naming is hard). That means "the input is not supported to do the aggregation by trivial operations (comparison, sum, etc) but require special implementations".